string                  cmd
geometry_msgs/Transform end_trans

---

geometry_msgs/Transform end2cam_trans

